### DISCUSSION TITLE
Remap for double button press

### DISCUSSION THREAD

**Discussion by priamos1** [cathery/sys-con#121] [Feature requests]
Is there a way to remap a function to a double button press? For instance, pressing both trigger buttons together to result in capture. As it stands I'm having trouble mapping something to capture for my xbox 360 controller WITHOUT unmapping anything else. I was using the left stick press, which is an example provided by the sys-con creator, but in some games that ends up being pretty bad. 

If there isn't a way, can this be added?

**Comment by cathery** [comment]
There's no way to do it right now, but I can add it. I've been considering adding accessibility features to sys-con, even though this is outside of scope of the sysmodule. But seeing as this is the only "controller" homebrew right now, i might push an update sometime in the future addressing these requests.
